modules/Movecraft/src/main/java/net/countercraft/movecraft/listener/InteractListener.java

```diff
@@ -114,21 +126,12 @@ public void onPlayerInteractStick(PlayerInteractEvent event) {
             // Player is onboard craft and right clicking
             float rotation = (float) Math.PI * event.getPlayer().getLocation().getYaw() / 180f;
 
-            float nx = -(float) Math.sin(rotation);
-            float nz = (float) Math.cos(rotation);
+            final Vector direction = player.getLocation().getDirection();
```

Comment:
is this normalized by default?

Reply:
Yes, direction vector is normalized by default